clang/lib/3C/ConstraintVariables.cpp

```diff
@@ -763,6 +771,16 @@ std::string PointerVariableConstraint::mkString(Constraints &CS, bool EmitName,
     EmittedName = true;
   uint32_t TypeIdx = 0;
 
+  // If we've set a GenericIndex for void, it means we're converting it into
+  // a generic function so give it the default generic type name.
+  // Add more type names below if we expect to use a lot.
+  std::string BaseName = BaseType;
```

Comment:
This could use a better name. `BaseName` makes me think it's the variables identifier. Maybe you can just update BaseType instead of creating the new var.

Reply:
`BaseType` is a CV field, and `mkString` may be called repeatedly, so I am hesitant to change it. But you're right that it isn't needed in the rest of the function, and at this point we've committed to the new generic name. Should I try `BaseTypeName` or do you still it's fine to just change `BaseType`?